static/app/components/events/interfaces/sourceMapsDebuggerModal.tsx

```diff
@@ -520,10 +521,10 @@ export function SourceMapsDebuggerModal({
               'An image says more than a thousand words. Below you can see a comparison of how a bad and a good stack trace look like:'
             )}
           </p>
-          <Fragment>
+          <div>
             <ContentSliderDiff.Header>
-              <ContentSliderDiff.BeforeLabel />
-              <ContentSliderDiff.AfterLabel />
+              <Flex align="center">{t('Without Source Maps')}</Flex>
+              <Flex align="center">{t('With Source Maps')}</Flex>
             </ContentSliderDiff.Header>
```

Comment:
@ryan953 I hope you don't mind. I pushed a commit to update the label and add a div in the source maps modal.

it looks like this now :

![image](https://github.com/user-attachments/assets/a1da54b6-4b02-4e6d-b95e-eeea7f3abe1d)